### PR TITLE
117d: Fixes changing code, closing WI, push to git and review and publish s…

### PR DESCRIPTION
**Pull request by Preeticp**
…ections.

### CODE REVIEW

File: docs/topics/modules/closing_workitem.adoc [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 
-. Click the btn:[Plan] tab and then btn:[Board] in the options to view the Planner Board view.
-. Drag the Work Item to the `Resolved` column.
+. Click the *Plan* tab and then *Board* to see the Planner Board view.
+. Drag the Work Item to the *Resolved* state.
 +
 image::resolved_wi.png[Resolved Work Item]
 
-Closing or resolving a Work Item communicated to the team that the task being tracked is complete.
+Resolving a Work Item communicates to the team that the task being tracked is complete. You can drag it to the *Closed* state, after it is reviewed and closed.
```
Comment (rkratky) [inline docs/topics/modules/closing_workitem.adoc]: This sounds strange: "You can drag it to the *Closed* state, after it is reviewed and closed." -- So, you close it (how?), and then you can drag it to 'Closed'?

I would expect that you close it **by** dragging it to 'Closed'.

File: docs/topics/modules/commit_push_changes_git.adoc [hunk 2]
```diff
@@ -3,19 +3,17 @@
 
 After making the required changes, commit and push the changes to your project GitHub repository:
 
-IMPORTANT: Before committing your changes, ensure that your project pipeline build has completed (i.e. the build has been promoted and is at the `Rollout to Run` stage). See <<approving_build_pipeline>> for instructions.
+IMPORTANT: Before committing your changes, ensure that your project pipeline build has completed (that is the build has been promoted and is at the `Rollout to Run` stage). See <<approving_build_pipeline>> for instructions.
```
Comment (rkratky) [inline docs/topics/modules/commit_push_changes_git.adoc]: -> "that is, the"

File: docs/topics/modules/review_publish_changes.adoc [hunk 3]
```diff
@@ -16,7 +16,7 @@ image::imp_build_2.png[Build #2 Runs]
 endif::[]
 //end conditional
 +
-. Click btn:[Create] and then btn:[Applications]. You can see the following information on this screen:
+. In the *Create* tab, switch to the *Applications* view, to see the following information:
```
Comment (rkratky) [inline docs/topics/modules/review_publish_changes.adoc]: -> "view to see"

File: docs/topics/modules/review_publish_changes.adoc [hunk 4]
```diff
@@ -28,15 +28,15 @@ image::imp_versions_applications.png[Versions of the Application]
 endif::[]
 //end conditional
 +
-** Different versions of your application are deployed to `Stage` and `Run`. Version 1.0.2 of the application is deployed to `Stage` because you have not yet promoted it to `Run`. The older version 1.0.1 is deployed to `Run` because you approved it last time the pipeline build executed.
+** Different versions of your application are deployed to *Stage* and *Run*. Version 1.0.2 of the application is deployed to *Stage* because you have not yet promoted it to *Run*. The older version 1.0.1 is deployed to *Run* because you approved it last time the pipeline build executed.
```
Comment (rkratky) [inline docs/topics/modules/review_publish_changes.adoc]: -> "version, 1.0.1, is"
-> "it the last time"

File: docs/topics/modules/review_publish_changes.adoc [hunk 5]
```diff
@@ -28,15 +28,15 @@ image::imp_versions_applications.png[Versions of the Application]
...
 ** The green check marks indicate that both builds are operational.
-** The number 1 indicates that each of the application builds is scaled to 1 pod in OpenShift Online. The number of pods indicates the number of running instances of the application.
-** The version numbers link to individual running applications. If you mouse over the version numbers, you will see a message about the environments. Click the version numbers to see the relevant application version running in `Stage`. You can share these URLs to compare the old and new versions of the application.
+** The number 1 indicates that each of the application builds is scaled to one pod in OpenShift Online. The number of pods indicate the number of running instances of the application.
```
Comment (rkratky) [inline docs/topics/modules/review_publish_changes.adoc]: -> "of pods indicates the number"

**Comment by Preeticp** [comment]
I think we could merge this too. All the comments have been addressed.

**Pull request merged**